server/test/repository/ApplicationStatusesRepositoryTest.java

```diff
@@ -79,20 +80,22 @@ public void canQueryForAllApplicationStatuses() {
 
     assertThat(statusDefinitionsModelResults).isNotEmpty();
     // one status is added as part of the program creation and one status as obsolete status
-    assertThat(statusDefinitionsModelResults.size()).isEqualTo(3);
+    assertThat(statusDefinitionsModelResults.size()).isEqualTo(4);
```

Comment:
question (non-blocking): If I'm understanding this correctly, we are still adding two statuses like we were previously (REAPPLY and APPROVED). Why are there 4 statuses instead of 3 now?

Reply:
+1 also curious about this

Reply:
I have reached out in the Slack Channel - I am puzzled too

Reply:
Maybe its worth checking for each whether they're live or obsolete and the name of the status definitions? And / or printing a debug string of `appStatusRepo.lookupAllApplicationStatusesModels( programDefinition.adminName())`?

Reply:
Hi @dkatzz - both are empty. Now I am sure the builder is getting called twice. 

 *********************************************In the builder 1
*********************************************In the builder Current active status -  []
*********************************************In the builder 2
*********************************************In the builder Current active status -  []
********************************************* 2